app/src/main/java/com/example/calculator2/ActivityMainFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.calculator2
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.calculator2.databinding.ActivityMainBinding
+
+class ActivityMainFragment: Fragment(R.layout.activity_main) {
```

Comment:
если используется метод onCreateView, то можно использовать без аргументов, ты там будешь создавать вью для фрагмента
```suggestion
class ActivityMainFragment: Fragment() {
```